Source/WebCore/rendering/RenderBox.h

```diff
@@ -161,13 +161,13 @@ class RenderBox : public RenderBoxModelObject {
     inline LayoutUnit logicalLeftVisualOverflow() const;
     inline LayoutUnit logicalRightVisualOverflow() const;
 
+    // Negative values mean overflow is allowed; positive or zero limit overflow.
```

Comment:
There would be no need for this comment if the return value used `std::optional<>`.